products/caasp/main.pm

```diff
@@ -203,40 +203,101 @@ sub stack_init {
     barrier_create("DELAYED_NODES_ONLINE", $stack_delayed + 1);
 }
 
-if (get_var('STACK_ROLE')) {
-    # ==== CaaSP tests ====
-    if (check_var 'STACK_ROLE', 'controller') {
-        stack_init;
-        loadtest "support_server/login";
-        loadtest "support_server/setup";
+# === Test Scheduling === #
+
+### openSUSE Kubic ###
+# ================== #
+if (is_caasp('kubic')) {
+
+    # Cluster/Multimachine Tests
+    if (get_var('STACK_ROLE')) {
+        # No MM tests yet
     }
+
+    # Single node Tests
     else {
-        if (update_scheduled 'dup') {
-            loadtest 'caasp/shift_version', name => 'ver=dec';
+        # kubeadm testsuites
+        if (check_var 'SYSTEM_ROLE', 'kubeadm') {
+            # Scenario: ISO Installation
+            if ((check_var 'SCENARIO', 'installation') && (check_var 'FLAVOR', 'DVD') && (check_var 'EXTRA', 'single_kubeadm')) {
```

Comment:
* the `EXTRA=single_kubeadm` was just an experiment.
* the `FLAVOR=DVD` is not required *atm*, but I would still like to have this redundant check in `main.pm` for the future. We plan to have `qcow2` images for kubic, and I this will *pro-actively* make things clearer. when the time has come. Anyway's it doesn't harm.